transform/transformers/PCKTransformer.py

```diff
@@ -117,11 +114,20 @@ def populate_period_data(self):
                 end_date = datetime.strptime(self.response['metadata']['ref_period_end_date'], "%Y-%m-%d")
                 self.data['12'] = end_date.strftime("%d/%m/%Y")
 
+    def round_currency_values(self):
```

Comment:
Rather then pre-processing en-block with hardcoded question numbers, would it be better to define "rounded integer" or similar as a `type` in the survey definition that can then be formatted as and when encountered? (Just as we have `checkbox` etc and when nothing is supplied the type is assumed to be `integer` (I believe))

Reply:
Agreed... but we want this in for Monday.  We can raise a card and get this looked at in the future because it's a change we'll probably want.